modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js

```diff
@@ -87,10 +88,15 @@ function ManagementToolbar({
 		() => normalizeDropdownItems(viewTypeItems),
 		[viewTypeItems]
 	);
-	const viewTypeIcon = useMemo(
-		() => viewTypeItems?.find((item) => item.active)?.icon,
+	const activeViewType = useMemo(
+		() => viewTypeItems?.find((item) => item.active),
 		[viewTypeItems]
 	);
+	const viewTypeIcon = activeViewType.icon;
+	const viewTypeTitle = sub(
```

Comment:
This is being used under some conditions (!active) so we may be executing it even when we don't need to. We probably could split the component so we don't have that massive JSX and improve this kind of things, but since this was the previous behavior of `viewTypeIcon` too and that I have the feeling (don't know why) that we need to FINISH THIS ASAP, just leave it as it is, just wanted to point it out.

Reply:
This can be an "improvement ticket" for this module and include the variable reported above